lib/index.js

```diff
@@ -17,50 +18,61 @@
   }
 
   function isOriginAllowed(origin, allowedOrigin) {
+    debug('test of origin %s. AllowedOrigin: %O',  origin, allowedOrigin);
```

Comment:
You can likely simply this entire block by creating a new variable to store the `true`/`false` and then have a single `debug` call at the end with what was checked and the result. This would also fix the issue here where given an array, it will never log `isAllowedOrigin false`.